src/components/z-status-tag/index.tsx

```diff
@@ -30,9 +30,10 @@ export class ZStatusTag {
   render() {
     return (
       <Host
-        class={classNames(this.type, this.variant, {
+        class={classNames(this.variant, {
           expandable: this.expandable && this.icon && this.text,
         })}
+        status={this.status}
```

Comment:
this is not necessary, `{ reflect: true }` does this for you

Reply:
Usando il tag Host nel render il reflect non funziona se si scrive direttamente il tag del web component. Facciamo altre prove, casomai togliamo reflect

Reply:
risolto